cmd/common.go

```diff
@@ -75,6 +76,20 @@ func AddCommonFlags(cmd *cobra.Command, cfg *CommonConfig) {
 	}
 }
 
+var (
+	clusterIdentifier = prometheus.NewGaugeVec(
```

Comment:
This is the metric we (GoCardless) have been using to join various time series together. By having all components report this in a consistent fashion (see [here](https://github.com/gocardless/stolon-pgbouncer/blob/master/cmd/stolon-pgbouncer/main.go#L132-L138) for the stolon-pgbouncer definition) it becomes possible to join series from totally different process/infrastructures on the cluster_name and store_prefix labels.